rules/terraformrules/terraform_module_pinned_source.go

```diff
@@ -56,8 +56,10 @@ func (r *TerraformModulePinnedSourceRule) Check(runner *tflint.Runner) error {
 
 		lower := strings.ToLower(module.SourceAddr)
 
-		if reGithub.MatchString(lower) || reBitbucket.MatchString(lower) || reGenericGit.MatchString(lower) {
+		if reGithub.MatchString(lower) || reGenericGit.MatchString(lower) {
 			r.checkGitSource(runner, module)
+		} else if reBitbucket.MatchString(lower) {
+		    r.checkBitbucketSource(runner, module)
```

Comment:
FYI, [gofmt](https://golang.org/cmd/gofmt/) is helpful for formatting your code. In many Go projects (including TFLint), it adopts gofmt to avoid style discussions.

Reply:
Thanks, done!